docs/backends.rst

```diff
@@ -44,8 +44,7 @@ It also works out of the box, because the wgpu-native DLL is shipped with wgpu-p
 
 The wgpu_native backend provides a few extra functionalities:
 
-.. py:function:: wgpu.backends.wgpu_native.request_device_sync(adapter, trace_path, *, label="", required_features, required_limits, default_queue)
-
+.. py:function:: wgpu.backends.wgpu_native.extras.request_device_sync(adapter, trace_path, *, label="", required_features, required_limits, default_queue)
```

Comment:
These functions are defined in `extras` but can be imported from `from wgpu.backends.wgpu_native`.